packages/create-webpack-app/templates/init/default/webpack.config.js.tpl

```diff
@@ -0,0 +1,101 @@
+// Generated using webpack-cli https://github.com/webpack/webpack-cli
+
+const path = require('path');<% if (htmlWebpackPlugin) { %>
+const HtmlWebpackPlugin = require('html-webpack-plugin');<% } %><% if (extractPlugin !== 'No') { %>
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');<% } %><% if (workboxWebpackPlugin) { %>
+const WorkboxWebpackPlugin = require('workbox-webpack-plugin');<% } %>
+
+const isProduction = process.env.NODE_ENV === 'production';
+<% if (cssType !== 'none') { %>
+<% if (extractPlugin === "Yes") { %>
+const stylesHandler = MiniCssExtractPlugin.loader;
+<% } else if (extractPlugin === "Only for Production") { %>
+const stylesHandler = isProduction ? MiniCssExtractPlugin.loader : 'style-loader';
+<% } else { %>
+const stylesHandler = 'style-loader';
+<% } %>
+<% } %>
+
+const config = {
+    entry: '<%= entryPoint %>',
```

Comment:
this should resolve to `src/index.js` in case of https://github.com/webpack/webpack-cli/pull/4167#issuecomment-2169066293

Reply:
Working on this